### PR TITLE
alive. PING command

### PR DESCRIPTION
**Pull request by ip75**
Send PING command to redis to check it is alive.

### CODE REVIEW

File: Redis.cpp [hunk 1]
```diff
@@ -715,6 +715,12 @@ RedisSubscribeResult Redis::startSubscribingNonBlocking(RedisMsgCallback message
   return RedisSubscribeSuccess;
 }
 
+bool Redis::alive()
```
Comment (rpj) [inline Redis.cpp]:
```suggestion
bool Redis::ping()
```

Unless absolutely necessary we'd prefer to keep methods that invoke Redis commands named the same as the Redis command they invoke.

File: Redis.h [hunk 2]
```diff
@@ -597,6 +597,11 @@ class Redis
 
   // auxiliary functions
 
+  /**
+   * Send PING command to redis to check is it alive
+  */
+  bool alive();
```
Comment (rpj) [inline Redis.h]:
```suggestion
  bool ping();
```

File: Redis.h [hunk 3]
```diff
@@ -597,6 +597,11 @@ class Redis
 
   // auxiliary functions
 
+  /**
+   * Send PING command to redis to check is it alive
```
Comment (rpj) [inline Redis.h]:
```suggestion
   * Send PING command to redis to check if it is alive
```

**Review by rpj — CHANGES_REQUESTED** [review]
Hi @ip75,

Thank you so much for your contribution, and welcome to the project!

There are a couple changes - just to naming - that I'd like to see with the code, plus I'd ask you to [add a test](https://github.com/electric-sheep-co/arduino-redis/blob/master/CONTRIBUTING.md?plain=1#L35) for this added method to the test suite & make sure your code [passes](https://github.com/electric-sheep-co/arduino-redis/actions/runs/7065427255/job/19238188745?pr=78) all the current tests as well. If you have trouble getting the [test harness](https://github.com/electric-sheep-co/arduino-redis#test-harness) running on your local system, please reach out and I'd be happy to help.

Additionally, if you wouldn't mind adding a short description to this PR (currently it just says "%SUBJ%") I would appreciate it.

Thank you again!
-RPJ

**Comment by ip75** [comment]
Hello
I absolutely agree with your changes and I've corrected comment for pull request.

**Pull request closed without merging**